Add extra debug information to logging

nosami opened the pull request.

Therzok commented on main/external/fsharpbinding/MonoDevelop.FSharpBinding/Services/Extensions.fs:
```diff
@@ -238,11 +239,25 @@ module AsyncChoiceCE =
+
+    let inline private logWithThread f format =
+
+#if DEBUG
```
Can't we add this in logging service?

nosami replied:
It could be added there if it's wanted.

nosami replied:
The stopwatch elapsed time is misleading if there are multiple threads logging concurrently - I should just remove this part

Therzok replied:
Yeah, we should have timings in general, not just the F# addin. /Cc @slluis

slluis commented:
@nosami if you still need this, please resolve conflicts and merge.

nosami commented:
I removed the stopwatch code and rebased this twice now. The first time, there was an unrelated test failure, this time the Unit Tests build seems to be stuck in `Pending`
